.azure-pipelines/client.yml

```diff
@@ -80,11 +75,10 @@ jobs:
       - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
         # ComponentGovernance is currently unable to run on pull requests of public projects. Running on non-PR
         # builds should be sufficient.
-        condition: and(succeededOrFailed(), ne(variables['Build.Reason'], 'PullRequest'))
+        condition: and(always(), ne(variables['System.TeamProject'], 'public'))
```

Comment:
Why is this condition being changed?  How is the new condition better or more correct?

Reply:
We should standardize on the condition currently in this yaml:

```
condition: and(succeededOrFailed(), ne(variables['Build.Reason'], 'PullRequest'))
```

It's better than the proposed change for 2 reasons:

1. `succeededOrFailed()` is better than `always()` because the latter runs even if pipeline is cancelled which we don't want.
2. Excluding on PRs is better than excluding public builds, since the former will still run on public builds of master.  Ideally we'd run CG on PRs as well, but it fails on fork PRs.  The best condition would actually disable fork PRs but run on non-fork PRs, but not sure how to implement that.

Reply:
Right, so for now let's standardize this condition itself. I can investigate to see how to implement that condition to disable forked PRs. For now I am closing this pull request and opening another one for updating the job name